packages/frontend/src/ui/date-time-picker/styles.module.css

```diff
@@ -54,5 +59,9 @@
 }
 
 .cellSelected {
-    @apply bg-gray-200;
+    @apply bg-gray-200 !text-black;
+}
+
+.cellSelectable {
+    @apply text-gray-300;
```

Comment:
Is this how it is in the design, if that's the case I think we have to change the color for the available hours and minutes, it's brighter than the disabled one, and it feels of, at least personally:
![image](https://github.com/user-attachments/assets/4d55ebae-79f1-4f60-95de-57803b6fa9b0)

Reply:
I agree :)